include/cpp2util.h

```diff
@@ -368,7 +368,7 @@ constexpr auto gcc_clang_msvc_min_versions(
 }
 
 
-#if defined(_MSC_VER)
+#if defined(_MSC_VER) && !defined(__clang_major__) && !defined(__GNUC__)
```

Comment:
Just checking `__clang_major__` should be enough. It's already done that way line 1465.
```suggestion
#if defined(_MSC_VER) && !defined(__clang_major__)
```